xlators/mgmt/glusterd/src/glusterd-op-sm.c

```diff
@@ -2830,6 +2832,151 @@ glusterd_set_shared_storage(dict_t *dict, char *key, char *value,
     return ret;
 }
 
+// Compare 2 strings for boolean value
```

Comment:
This is not correct. The function compares a string (normally, expression of a Boolean) with an interger (normally, a Boolean value).